src/app/pages/blog/article/article.tsx

```diff
@@ -220,7 +220,6 @@ function ShareButtons() {
             message={encodeURIComponent(
                 'Check out this OpenStax blog article!'
             )}
-            minimal={true}
```

Comment:
The share component is only being used in in this place, which used the `minimal` option, so I made it not an option.